hedera-node/hedera-consensus-service-impl/src/main/java/com/hedera/node/app/service/consensus/impl/handlers/ConsensusCreateTopicHandler.java

```diff
@@ -81,6 +85,18 @@ public void handle(@NonNull final HandleContext handleContext) {
         requireNonNull(handleContext, "The argument 'context' must not be null");
 
         final var op = handleContext.body().consensusCreateTopicOrThrow();
+
+        final var fees = handleContext.feeCalculator(SubType.DEFAULT).legacyCalculate(sigValueObj -> {
+            try {
+                final var protoBody = fromPbj(handleContext.body());
+                return ConsensusServiceFeeBuilder.getConsensusCreateTopicFee(protoBody, sigValueObj);
+            } catch (InvalidTxBodyException e) {
+                throw new RuntimeException(e);
```

Comment:
It might be good to define an actual interface (perhaps as an inner class on the FeeCalculator interface) for the legacy callback, so that the method can throw a HandleException rather than just a raw RuntimeException.  That way we get a proper controlled status code, rather than internal error response.

Reply:
In the mono service, it does something similar:

```
            try {
                ...
            } catch (InvalidTxBodyException e) {
                log.warn(
                        "Argument accessor={} malformed for implied estimator {}!",
                        accessor.getSignedTxnWrapper(),
                        usageEstimator);
                throw new IllegalArgumentException(e);
            }
```

To maintain compatibility, we will need to also through a runtime exception rather than a HandleException. When we replace this with the new fee implementation, all of these errors will go away naturally and not be possible to throw (since we actually deal with this body checking when we get it from the context)